slam-ws/src/nav_stack/launch/move_base.launch

```diff
@@ -58,6 +58,8 @@
 
       <!-- Rename cmd_vel topic name from move_base -->
       <remap from="cmd_vel" to="nav_vel"/>
+
+      <remap from="map" to="rtabmap/grid_map"/>
```

Comment:
it might be better to remap the output of rtab map occupancy grid from `rtabmap/grid_map` to `map` so we can still use rtab and gmapping with the same `move_base` configuration